graphium/fingerprinting/data.py

```diff
@@ -0,0 +1,216 @@
+from typing import Any, List, Dict, Literal, Union
+
+import os
+
+import torch
+
+import pandas as pd
+import numpy as np
+
+from pytorch_lightning import LightningDataModule
+
+from torch.utils.data import Dataset, DataLoader
+
+from graphium.data.datamodule import MultitaskFromSmilesDataModule, ADMETBenchmarkDataModule
+from graphium.trainer.predictor import PredictorModule
+from graphium.fingerprinting.fingerprinter import Fingerprinter
+
+
+class FingerprintDataset(Dataset):
```

Comment:
Missing docs for parameters of that class.

Reply:
Fixed [here](https://github.com/datamol-io/graphium/pull/529/commits/d9aa407d774934e170e8d8af7028155b676b5981#diff-a660520d63c09919ce7b7d4450b3edf709bba2c5d21fa421a52fa4d1fff7a9e3).